### DISCUSSION TITLE
What's the mean of group_size of psROIPooling on MxNet?

### DISCUSSION THREAD

**Discussion by elbert-xiao** [apache/incubator-mxnet#19699] [Q&A]
I see a usage of psroipooling in the code of **FCIS** paper:
`psroipool_cls_seg = mx.contrib.sym.PSROIPooling(name='psroipool_cls_seg', data=fcis_cls_seg, rois=rois,
                                                        group_size=7, pooled_size=21, output_dim=num_classes*2, spatial_scale=0.0625)`
I can understand the 'pool_ size' is the number of bins, but what's the mean of 'group_size'?

**Comment by elbert-xiao** [comment]
This question has been resolved！
The group_size can be understood as the window size of the feature map from each channel.

**Reply by DingZhangIntel** [reply]
Hi, i have the same problem about this group_size, please help me to understand the mean of 'gourp_size' at any your convenience, maybe a example can do the better effect, thanks.
